packages/website/versioned_docs/version-latest/extending-cadl/linters.md

```diff
@@ -24,7 +24,7 @@ npm install @cadl-lang/lint
 ### 2. Define the rules
 
 ```ts
-import { createRule } from "@cadl-lang/lint";
```

Comment:
@allenjzhang can you confirm--am I supposed to update files in this tree or leave them alone?

Reply:
This folder is updated by the publish step. I think you should revert this.

Reply:
You should leave `/versioned_docs/**/*` untouched. This folder will be bulk updated with snapshot of /docs when next publish (hopefully today) goes out. 

Did you update this file manually? All doc changes should be made under /docs.